### PR TITLE
Add volcano integration docs

### PR DESCRIPTION
**Pull request by TommyLike**
Add documents on volcano integration.

For #583

### CODE REVIEW

**Comment by TommyLike** [comment]
@k82cn, @liyinan926  please review when you have time.

File: docs/volcano-integration.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Integrate with volcano scheduler for Spark Application
+
+[Volcano](https://github.com/volcano-sh/volcano) is a batch system built on Kubernetes. It provides a suite of mechanisms
+currently missing from Kubernetes that are commonly required by many classes
+of batch & elastic workload.
```
Comment (liyinan926) [inline docs/volcano-integration.md]: `workload` -> `workloads`.

File: docs/volcano-integration.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+Now the spark application pods can be scheduled via volcano for better scheduling efficiency.
```
Comment (liyinan926) [inline docs/volcano-integration.md]: I suggest rephrasing this to `With the integration with Volcano, Spark application pods can be scheduled ...`.

File: docs/volcano-integration.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+
+# Requirements
+
+## Volcano components
+
+Before using spark-operator with volcano enabled, user need to ensure volcano has been successfully installed in the
```
Comment (liyinan926) [inline docs/volcano-integration.md]: s/`spark-operator`/`Kubernetes Operator for Apache Spark`. This is a requirement for this project to be listed on the Spark website.

File: docs/volcano-integration.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+same environment, please refer [quick start guide](https://github.com/volcano-sh/volcano#quick-start-guide) for volcano installation.
```
Comment (liyinan926) [inline docs/volcano-integration.md]: `please refer to the [Quick Start Guide]`.

File: docs/volcano-integration.md (hunk 3 above)
Comment (liyinan926) [inline docs/volcano-integration.md]: Let's use `Volcano` (with the first letter capital) consistently throughout the doc.

File: docs/volcano-integration.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+
+Within the help of helm chart, spark operator with volcano can be easily installed with the command below:
+```bash
+$ helm repo add incubator http://storage.googleapis.com/kubernetes-charts-incubator
+$ helm install incubator/sparkoperator --namespace spark-operator --set enableBatchScheduler=true
+```
+
+# Run Spark Application with volcano scheduler
+
+Now, we can run a updated version of spark application (with `batchScheduler` configured), for instance:
```
Comment (liyinan926) [inline docs/volcano-integration.md]: Let's use `Spark` (with the first letter capital) throughout the doc.

File: docs/volcano-integration.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+# Requirements
+
+## Volcano components
+
+Before using spark-operator with volcano enabled, user need to ensure volcano has been successfully installed in the
+same environment, please refer [quick start guide](https://github.com/volcano-sh/volcano#quick-start-guide) for volcano installation.
+
+## Install spark operator with volcano enabled.
+
+Within the help of helm chart, spark operator with volcano can be easily installed with the command below:
```
Comment (liyinan926) [inline docs/volcano-integration.md]: s/`spark operator`/`Kubernetes Operator for Apache Spark`.

Comment (liyinan926) [inline docs/volcano-integration.md]: s/`helm`/`Helm`.

**Comment by TommyLike** [comment]
@liyinan926  done!

File: docs/volcano-integration.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
+# Integrate with Volcano scheduler for Spark Application
```
Comment (liyinan926) [inline docs/volcano-integration.md]: I think `Integration with Volcano for Batch Scheduling` is a better title.

File: docs/volcano-integration.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+of batch & elastic workloads.
+With the integration with Volcano, Spark application pods can be scheduled for better scheduling efficiency.
+# Requirements
+
+## Volcano components
+
+Before using Kubernetes Operator for Apache Spark, with Volcano enabled, user need to ensure Volcano has been successfully installed in the
+same environment, please refer [Quick Start Guide](https://github.com/volcano-sh/volcano#quick-start-guide) for Volcano installation.
+
+## Install Kubernetes Operator for Apache Spark with Volcano enabled.
```
Comment (liyinan926) [inline docs/volcano-integration.md]: No period at the end of titles/subtitles.

**Pull request merged**